src/main/java/com/onevizion/scmdb/SqlScriptExecutor.java

```diff
@@ -50,6 +56,19 @@ public SqlScriptExecutor() {
         executor.setStreamHandler(new PumpStreamHandler(new LogOutputStream() {
             @Override
             protected void processLine(String line, int logLevel) {
+                if (line.contains(INVALID_OBJECT_IN_SCHEMA)) {
```

Comment:
This solution looks unstable, please review 'compile_invalids_wrapper_fail_on_error.sql' and provide more reliable one

Reply:
New way.
Found a system variable disabling result headers. Now output 20 equals 19.